src/config.rs

```diff
@@ -527,10 +527,13 @@ impl Config {
 
 fn find_global_config_file() -> Result<PathBuf> {
     dirs::home_dir()
-        .ok_or_else(|| format_err!("Could not find home dir"))
-        .map(|mut path| {
-            path.push(CONFIG_RC_FILE_NAME);
-            path
+        .map(|p| p.join(CONFIG_RC_FILE_NAME))
+        .filter(|p| p.exists())
+        .or(dirs::config_dir()
+            .map(|p| p.join("sentry/sentrycli.ini"))
+            .filter(|p| p.exists()))
+        .ok_or_else(|| {
+            format_err!("Could not find config file. Please run `sentry-cli login` and try again!")
```

Comment:
```suggestion
            format_err!("Could not find config file.")
```

Why add the note about running `sentry-cli login` here? I am unsure whether this is always necessary – auth token is not needed for all CLI requests. Even if we should be adding this, this change should be made in a separate PR, since it is unrelated to what we want to do in this change.